.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.pdf -text
+*.rb eol=lf
```

Comment:
I really dislike that fact that we have to have a .gitattributes file. Is it possible to configure these globally in GitHub Action's Windows runner?

Reply:
I observe exactly the same test failures on my Windows machine. Git for Windows has `core.crlf=true` by default, so all text files are checked out to working copy with CRLF line endings.

Why you're against .gitattributes? It is just a tool to configure Git behavior.

Reply:
Because I believe it should be configured globally. If you add CRLF newlines to source code, you are not running the same source code. Therefore, I see no reason why anyone should ever have this enabled.

Reply:
That's what `*.rb eol=lf` does! It says "whoever checks out *.rb files, gets them with LF line endings".

Reply:
To me, it's a stain on the project. I think it should be documented in the README instead. If you want to develop on this project, you can't have that setting. It may be a feature of git, but it should not be used IMO.

Reply:
As I already said, it is current [default system-wide setting in Git for Windows](https://github.com/git-for-windows/build-extra/blob/master/git-extra/git-extra.install#L9). And there are cases where this is a desired behavior.

I don't understand why you want people to perform manual actions to do what can be achieved with a two-line config file.

Reply:
Because first of all, it's not complete. If we were going to do it, we need to address every file the repository. And frankly, that's just a pain. I'll keep saying it. There's no reason to do endline conversion in a development environment. We don't reencode from UTF-8 to other encodings automatically. So why would we touch endlines. That is not a problem the project should have to deal with. That is the responsibility of the developer.

Reply:
I split eol issue into #1523.